integration-tests/container/perf/Dockerfile

```
@@ -1,17 +1,6 @@
-FROM alpine:3.12 AS base
-
-RUN apk add --update \
-    bash \
-    bc \
-    build-base \
-    bison \
-    flex \
-    curl \
-    elfutils-dev \
-    linux-headers \
-    make \
-    openssl-dev \
-    diffutils
+FROM quay.io/centos/centos:stream9 AS base
+
+RUN dnf install -y bash bc bison curl-minimal diffutils elfutils-devel flex kernel-devel make openssl-devel
```

[nit] can we split these dependencies on a line each

```suggestion
RUN dnf install -y \
    bash \
    bc \
    bison \
    curl-minimal \
    diffutils \
    elfutils-devel \
    flex \
    kernel-devel \
    make \
    openssl-devel
```

---

yes, tested on z ... @pratham-m can you test on power?

---

verified that the images builds fine on ppc64le also